packages/lib/src/dropdown/DropdownMenuItem.tsx

```diff
@@ -3,6 +3,57 @@ import styled from "styled-components";
 import { DropdownMenuItemProps } from "./types";
 import DxcIcon from "../icon/Icon";
 
+const DropdownMenuItemContainer = styled.li<{ visuallyFocused: DropdownMenuItemProps["visuallyFocused"] }>`
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  gap: var(--spacing-gap-xs);
+  min-height: 36px;
+  padding: 0px var(--spacing-padding-xs);
+  cursor: pointer;
+
+  ${(props) =>
+    props.visuallyFocused &&
+    `
+  outline: var(--border-width-m) solid var(--border-color-secondary-medium); 
+  outline-offset: calc(-1 * var(--border-width-m));
+`}
+  &:first-child {
+    border-top-left-radius: var(--border-radius-s);
+    border-top-right-radius: var(--border-radius-s);
+  }
+
+  &:last-child {
+    border-bottom-left-radius: var(--border-radius-s);
+    border-bottom-right-radius: var(--border-radius-s);
+  }
+  &:hover {
+    background-color: var(--color-bg-neutral-light);
+  }
+  &:active {
+    background-color: var(--color-bg-neutral-light);
+  }
+`;
+
+const DropdownMenuItemLabel = styled.span`
+  font-family: var(--typography-font-family);
+  font-size: var(--typography-label-l);
+  font-weight: var(--typography-label-regular);
+  color: var(--color-fg-neutral-dark);
+  white-space: nowrap;
+`;
+
+const DropdownMenuItemIcon = styled.div`
+  display: flex;
+  color: var(--color-fg-neutral-dark);
+  font-size: var(--typography-label-xl);
```

Comment:
Same as before, I suggest to use `--height-xs` instead of the current `--typography-label-xl`.